### DISCUSSION TITLE
Speed Comparison of Molecular Simulations: MACE (or NequIP) in DP's LAMMPS Plugin vs Their Provided LAMMPS Plugins (Including Kokkos Usage)

### DISCUSSION THREAD

**Discussion by clecust** [deepmodeling/deepmd-kit#4427] [Q&A]
Hello everyone,
I am currently working on molecular simulations and am very concerned about the speed of molecular simulations .
The problem is as stated in the title.

I would really appreciate any insights or suggestions from the community. Thanks in advance!

**Comment by njzjz** [comment]
Just make a note here (not an answer): currently the developer team has nothing to share.
